### DISCUSSION TITLE
Profiles and Repositories

### DISCUSSION THREAD

**Discussion by 86ul** [borgbase/vorta#1830] [FAQ] [answered]
Hello fellow users of Vorta for Borg BackUp,

Recently, I noticed that when I change Profile, the Repository does _**not**_ whith it. Is this a normal behaviour?

For example, I have set two profiles. Profile one is called Daily for 'hot' hourly backups, and profile two is called Weekly for 'cold' weekly backups. After putting two different repositories in each profile, with the result that each profile has their own unique repository and settings, I noticed that the repository does not change, when I change the profile. 

Is this a normal behaviour? If yes, how do you advice me to set up Vorta? If no, shall I report this issue as a bug?

Thanks,

86ul

**Comment by m3nu** [comment, marked as answer]
2 profiles can use the same repo. But generally all settings except _Misc_ are related to a profile and will change when changing the dropdown.

**Reply by 86ul** [reply]
merci
